### PR TITLE
Set wind rose text clickable + text size

### PR DESCRIPTION
**Pull request by fbunoz**
See https://github.com/c2corg/v6_ui/issues/1344

Before : 

rose_avant

After : 

rose_apres

### CODE REVIEW

**Comment by brunobesson** [comment]
@fbunoz sorry, I was working on the same subject in the meantime, along with other enhancements (see https://github.com/c2corg/v6_ui/tree/wind-rose). what I have not done yet is the text size.
Are you OK if I merge your commits into mine and discard your PR?

File: c2corg_ui/templates/helpers/orientations.svg [hunk 1]
```diff
@@ -24,9 +24,9 @@
     <path id="ori-ne" d="m369.46 166.83-141.65 59.371 59.368-141.65 0.002-0.002 100.34-18.058z"
       ng-click="${ctrl}.toggleOrientation('NE', ${model}, $event, '${filterName}')"
       ng-class="{ 'orientation-selected': ${model}.orientations.indexOf('NE') > -1 }"/>
-    <text id="text-n" y="99.007141" x="212.0424" font-size="40">N</text>
-    <text id="text-s" y="385.00714" x="214.10295" font-size="40">S</text>
-    <text id="text-w" y="241.58344" x="64" font-size="40">W</text>
-    <text id="text-e" y="241.58344" x="358" font-size="40">E</text>
+    <text id="text-n" y="100.007141" x="205.0424" font-size="60">N</text>
```
Comment (brunobesson) [inline c2corg_ui/templates/helpers/orientations.svg]: Y coordinate should be 105 instead for correct centering (I corrected it on my branch).

File: less/documentedit.less [hunk 2]
```diff
@@ -602,15 +602,17 @@ input[type="radio"] {
...
-    cursor: pointer;
+  path {
     transition: 0.3s ease-in-out;
 
     &.orientation-selected {
       transition: 0.3s ease-in-out;
       fill: @C2C-orange !important;
     }
   }
+  path, text {
```
Comment (brunobesson) [inline less/documentedit.less]: Changing the text cursor is not enough, the action must also be applied. One should also surround both path and text with a `g` tag. At least this is how I did in my branch.

**Comment by fbunoz** [comment]
> Are you OK if I merge your commits into mine and discard your PR?

Ok, it's better you discard my PR.

**Comment by brunobesson** [comment]
OK, closing then. Thanks for the font size corrections!

**Comment by asaunier** [comment]
@brunobesson Don't forget to delete the branch after closing the PR (I have done it).
Anyway @fbunoz thanks for the PR!

**Comment by brunobesson** [comment]
For safety, I was planning to remove the branch only after my own PR would be merged. But OK anyway. I also forgot to create the PR, which I just did: #1351

**Comment by asaunier** [comment]
The diff is still available in the PR even though the branch is deleted: https://github.com/c2corg/v6_ui/pull/1345/files

**Comment by brunobesson** [comment]
>The diff is still available in the PR

Didn't think of that, thanks for mentioning.

**Pull request closed without merging**